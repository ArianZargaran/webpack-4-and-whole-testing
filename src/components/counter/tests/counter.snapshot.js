import React from "react";
import renderer from "react-test-renderer";

import Counter, { CounterEl } from "../counter";

describe("Counter Snapshot", () => {
  test("renderers", () => {
    const component = renderer.create(
      <Counter />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe("CounterEl Snapshot", () => {
  test("renderers", () => {
    const component = renderer.create(
      <CounterEl counter={0} />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});