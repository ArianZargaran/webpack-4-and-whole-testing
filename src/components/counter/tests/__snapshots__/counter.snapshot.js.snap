// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter Snapshot renderers 1`] = `
<div>
  <h1>
    My Counter
  </h1>
  <p>
    0
  </p>
  <button
    onClick={[Function]}
    type="button"
  >
    Increment
  </button>
  <button
    onClick={[Function]}
    type="button"
  >
    Decrement
  </button>
</div>
`;

exports[`CounterEl Snapshot renderers 1`] = `
<p>
  0
</p>
`;
